checks/libs/win/winpdh.py

```diff
@@ -1,18 +1,44 @@
 import time
 import win32pdh
+import _winreg
 
 DATA_POINT_INTERVAL = 0.01
 
 class WinPDHCounter(object):
+    # store the dictionary of pdh counter names
```

Comment:
Could we add some more info into the comment about how the dictionary translates from "US-English" name to the index identifier for the counter?